app/views/articles/_sidebar_nav.html.erb

```diff
@@ -19,17 +19,17 @@
         navigation
       </div>
       <div class="sidebar-nav-element">
-        <a class="sidebar-nav-link" href="/readinglist?we-hope-you-like-the-new-and-improved-reading-list---more-positive-changes-coming=woohoo">
+        <a class="sidebar-nav-link" href="<%= readinglist_path %>">
```

Comment:
This refactoring is because I was going to add the href to the history page in the homepage but decided against it. Not sure if we should put the link for pro users here.